templates/description-list.html.twig

```diff
@@ -0,0 +1,19 @@
+{#
+/**
+ * @file
+ * Description list template.
+ */
+#}
+
+{% if items %}
+  <dl>
+    {% for item in items %}
+      {% if item.term %}
+        <dt>{{ item.term }}</dt>
```

Comment:
This doesn't really work. I left one term empty and it printed the <dt>:

<img width="306" alt="Screen Shot 2021-02-24 at 14 26 20" src="https://user-images.githubusercontent.com/5848933/109007305-493b2f00-76ac-11eb-8142-9322d6dd4a6a.png">

It means it's missing also some test coverage.